_data/sites.json

```diff
@@ -4676,6 +4676,16 @@
         ]
     },
 
+    {
+        "name": "Melodics",
+        "url": "https://melodics.com/support#form-header",
+        "difficulty": "hard",
+        "notes": "Use the form to contact the support team and wait for their reply to confirm.",
```

Comment:
```suggestion
        "notes": "It might be possible to delete your account via the contact form, but the support team is known to be unresponsive.",
```